### ISSUE TITLE
Current maintenance state

### ISSUE THREAD

**Issue by McShelby** [matcornic/hugo-theme-learn#442]
Hiya @matalo33, @matcornic, 

as already stated in an email conversation with @matcornic, this hugo theme is one of a kind and I really love it.

While working with the theme I suffered from a lot of issues other users already have reported. Also I could provide PRs for some of them and could help in general.

Sadly, new versions of the theme were sparse for the past year. Which is a pity because the theme IS really great and a lot of people are using it.

How is the current state of the project? I refuse to fork the project right now, because I prefer to put fixes into the official version. I can offer some help if you let me.

Thanks,
Sören

**Comment by matalo33** [comment]
Hi @McShelby, 

I'm sure everyone has noticed the review of PRs and handling of issues is very sporadic. I also really like the theme however I'm not able to commit much time to helping with maintenance at the moment. It is a very popular theme, it does deserve more attention than it gets.

I agree forking might be more harmful than good so this is a very welcome question. The repo belongs to @matcornic and as such the decision lies with him, but I'd be very happy to work together on maintenance.

Over to you @matcornic

**Comment by matalo33** [comment]
I had a chat with @matcornic the other day. Both of us are very happy to get additional help maintaining this theme however we'd like to see some supporting evidence of your application in the way of activity on the project (e.g. PR submissions, PR reviews/testing, etc.)

Were there any particular features you wanted to add?

Additionally my biggest difficulty at the moment is having time to test PRs before merging them. If you wanted to clone some yourself to run against your websites and provide feedback it would be a huge help.

**Comment by McShelby** [comment]
Sounds great! I have a view things in mind to work on in the begining. Especially issues that are already opened by other users and I discovered for myself. I'll prepare some PRs the next few days so both of you can see what I am working on.

I already read the wiki guidelines and have some ideas for 3.0 - but first things first...

**Comment by matcornic** [comment]
Hello everyone,

Yes, as said by Matthew, all help is welcome!
I'm not comfortable enough to set as maintainer someone I don't know personally and who has never contributed on this project though :). I hope you understand.

Nevertheless, I can't wait to see what you are preparing 👍

**Comment by McShelby** [comment]
I am not giving up on this. Maybe we could set up an Videomeeting on this to talk a bit. It's sad to see all those lingering PRs with some cool ideas getting stale without any feedback or progression. Unfortunatly at the moment this nice project seems dead - and that's a pitty!

**Comment by muffmolch** [comment]
I, too, would like to see a forced further development of this repository. In my opinion, there are some essential bug fixes in PR that just need to be adopted. But unfortunately the project has been at a standstill for almost a year :-(

**Comment by McShelby** [comment]
I've forked this project to [McShelby/hugo-theme-relearn](https://github.com/McShelby/hugo-theme-relearn). The fork is meant to be a drop-in replacement of this theme.

The first release 1.0.0 was on 2021-07-01. Version 2.0.0 of my fork was released a few hours ago. All together it [fixes 57 issues](https://github.com/McShelby/hugo-theme-relearn/issues?q=is%3Aissue+is%3Aclosed), many of them are long outstanding in this theme's issue list.

Forking the theme wasn't a light decision and loomed in me for almost a year. I've made proposals on how to revive the development multiple times but with no success. Even other users are discussing the theme's state to no success (see matcornic/hugo-theme-learn#502, matcornic/hugo-theme-learn#505). I've made PRs, code reviews and giving help in the issues comments with almost no reaction from the maintainers.

Although this repo mentions the last update to be on 2021-03-31, the commit history doesn't show significant development for almost a year.

I am more than willing to dump my fork if the development of the this theme comes up to speed again. **[But for the moment the learn theme is dead](https://github.com/matcornic/hugo-theme-learn/issues/348#issuecomment-581080113)**.
